rust-toolchain.toml

```diff
@@ -0,0 +1,5 @@
+[toolchain]
```

Comment:
Once we are done with the cleanup step, this file should be the source of truth for the toolchain/s to use